templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
```

Comment:
`keywords` and `description` meta tags are probably redundant, as they should already be added by`{{{ head.meta_tags }}}` (if you use the `empty` template) .